unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -408,17 +413,17 @@ export default class ActionBinder {
       await this.showSplashScreen();
       switch (e.status) {
         case 409:
-          await this.dispatchErrorToast('verb_upload_error_duplicate_asset');
+          await this.dispatchErrorToast('verb_upload_error_duplicate_asset', e.showError);
           break;
         case 401:
-          await this.dispatchErrorToast(e.message === 'notentitled' ? 'verb_upload_error_no_storage_provision' : 'verb_upload_error_generic');
+          await this.dispatchErrorToast(e.message === 'notentitled' ? 'verb_upload_error_no_storage_provision' : 'verb_upload_error_generic', e.showError);
           break;
         case 403:
-          if (e.message === 'quotaexceeded') await this.dispatchErrorToast('verb_upload_error_max_quota_exceeded');
-          else await this.dispatchErrorToast('verb_upload_error_no_storage_provision');
+          if (e.message === 'quotaexceeded') await this.dispatchErrorToast('verb_upload_error_max_quota_exceeded', e.showError);
+          else await this.dispatchErrorToast('verb_upload_error_no_storage_provision', e.showError);
           break;
         default:
-          await this.dispatchErrorToast('verb_upload_error_generic');
+          await this.dispatchErrorToast('verb_upload_error_generic', e.showError);
```

Comment:
Any reason for not using optional chaining here?

Reply:
I see e was already used here for checking e.message etc. without optional chaining hence assumed that e will always be defined.

Reply:
Ah yes my bad, this was a misunderstanding on my part but makes sense.